src/app/features/tech-record/components/tech-record-change-type/tech-record-change-type.component.html

```diff
@@ -1,56 +1,53 @@
-      <dt class="govuk-summary-list__key">VIN/chassis number</dt>
-      <dd id="vehicleType" class="govuk-summary-list__value">{{ vehicle.vin | defaultNullOrEmpty }}</dd>
-    </div>
-  </dl>
-
-  <form [formGroup]="form">
-    <app-select
-      #selectVehicleType
-      [label]="form.meta.children![0].label"
-      [formControlName]="form.meta.children![0].name"
-      [options]="vehicleTypeOptions"
-      name="change-vehicle-type-select"
-      (ngModelChange)="ngOnChanges()"
-    ></app-select>
-
-    <div *ngIf="selectVehicleType.value && selectVehicleType.value !== currentTechRecord?.vehicleType">
-      <h1 class="govuk-heading-xl">Are you sure you want to change this Vehicle type?</h1>
-
-      <p class="govuk-body">
-        You are about to change a {{ currentTechRecord?.vehicleType | uppercase }} to a {{ selectVehicleType.value | uppercase }} provisional
-        technical record.
-      </p>
-    </div>
-
-    <div class="buttons-row">
-      <button
-        class="govuk-button"
-        data-module="govuk-button"
-        type="submit"
-        enterkeyhint="change vehicle type"
-        (click)="handleSubmit(selectVehicleType.value)"
-      >
-        Confirm and continue
-      </button>
-
-      <app-button design="link" data-module="govuk-button" (clicked)="navigateBack()">Back</app-button>
-    </div>
-  </form>
-</div>
+<h2 class="govuk-heading-l">{{ makeAndModel }}</h2>
+
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">Vehicle type</dt>
+    <dd id="vehicleType" class="govuk-summary-list__value">
+      {{ currentTechRecord?.vehicleType | defaultNullOrEmpty | uppercase }}
+    </dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">VRM</dt>
+    <app-number-plate *ngIf="vrm" [vrm]="vrm"></app-number-plate>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">VIN/chassis number</dt>
+    <dd id="vehicleType" class="govuk-summary-list__value">{{ vehicle?.vin | defaultNullOrEmpty }}</dd>
+  </div>
+</dl>
+
+<form [formGroup]="form">
+  <app-select
+    #selectVehicleType
+    formControlName="selectVehicleType"
+    name="change-vehicle-type-select"
+    label="Select a new vehicle type"
+    [options]="vehicleTypeOptions"
+  ></app-select>
+
+  <ng-container *ngIf="selectVehicleType.value">
+    <h1 class="govuk-heading-xl">Are you sure you want to change this Vehicle type?</h1>
+
+    <p class="govuk-body">
+      You are about to change a {{ currentTechRecord?.vehicleType | uppercase }} to a {{ selectVehicleType.value | uppercase }} provisional technical
+      record.
+    </p>
+  </ng-container>
+
+  <div class="buttons-row">
+    <button
+      class="govuk-button"
+      data-module="govuk-button"
+      type="submit"
+      enterkeyhint="change vehicle type"
+      (click)="handleSubmit(selectVehicleType.value)"
+    >
+      Confirm and continue
+    </button>
```

Comment:
Not super important, but why not use the `app-button` here?

Reply:
I haven't written or updated anything on this template, just removed a level a unnecessary nesting.